website/content/docs/agent/options.mdx

```diff
@@ -64,6 +64,15 @@ The options below are all specified on the command-line.
   state as other nodes will treat the non-routability as a failure. In Consul 1.1.0 and later this can be dynamically defined with a [go-sockaddr]
   template that is resolved at runtime.
 
+  <CodeBlockConfig>
+
+  ```shell
+  # Using a static network interface name
+  $ consul agent -bind '{{ GetInterfaceIP "eth0" }}'
+  ```
+
+  </CodeBlockConfig>
+
```

Comment:
<CodeBlockConfig>

  ```shell
  # Using a static network interface name
  $ consul agent -advertise '{{ GetInterfaceIP "eth0" }}'
  ```

  </CodeBlockConfig>

I think you meant `-advertise` instead of `-bind`. Otherwise, this is already present at <https://www.consul.io/docs/agent/options#_bind>.

Reply:
Yep, that is what I meant.

Thanks for catching that!